old/bin/aa.pl

```diff
@@ -949,7 +949,8 @@ sub form_footer {
             $hold_text = $locale->text('On Hold');
         }
 
-
+# REMARK: According to the check below, post_as_new and delete button never render
+# Why post_as_new and delete are exist?
```

Comment:
Good point (asking why `delete` and `post_as_new` exist when they can't be triggered). However, I think that belongs either in a discussion item in GitHub, or in an issue, or even in the text that goes with the pull request. The *answer* to the question however, *does* belong in the code...

The answer to the question is as simple as disappointing: this code has had too many iterations by people who did not fully comprehend their changes, but did know just enough to be able to test that their edits worked (for their use-case). Transactions used to be deletable, but then "draft" transactions came and not all transactions could be deleted anymore. This required the Delete button to be (temporarily?) disabled. I think the same applies to "Post as New".